lib/internal/timers.js

```diff
@@ -37,6 +38,14 @@ function getTimers() {
   return timers;
 }
 
+function initAsyncResource(resource, type = 'Timeout') {
```

Comment:
Non-blocking nit: I personally would remove the default and always pass in the value explicitly.

Reply:
Sure, I was undecided on whether to do this or not so adjusted now.